databrowser/src/pages/DatasetListPage.vue

```diff
@@ -21,7 +21,7 @@
             :key="col.title"
```

Comment:
What do you think about moving this file into a sub directory? For example like `pages/datasets/List.vue`. Like this we don't have everything at the top level.

Reply:
I agree that this file needs to be relocated. Given that the list, detail an raw views are shown on the same page (switchable by the tab-like buttons "Table view", "Detail view" and "Raw data"), I would opt to keep something like `DatasetPage.vue` in the `pages` folder and to move the different views either into the `components` folder or to modularize it like you suggested and put it somewhere beneath the pages folder, e.g. `pages/datasets/...`.

If it's ok with you, I would do that refactoring in a later stage, I'm already working on the next PR where it could be part of (that PR contains improved table rendering and pagination).

Reply:
For me it is always hard to find files, if the pages folder does not follow the same structure as the components page. That's why I proposed it. But we can do this refactoring also in another PR.

Reply:
If there is no differing opinion I'm going to decompose the `DatasetListPage.vue` into several smaller components using the naming scheme you suggested. The change will be done on a subsequent PR.